tf/variables.tf

```diff
@@ -4,6 +4,11 @@ variable "aws_region" {
   default     = "eu-central-1"
 }
 
+variable "user_name" {
+  description = "Required username to use for prefixes"
+  type        = string
+}
+
```

Comment:
Should this variable be also removed? It doesn't seem to be used.

Reply:
I think it's used here: https://github.com/elastic/apm-aws-lambda/pull/355/files#diff-a2fd895ac25407846fd26b6d4fc9a4482f8cab6767431f76535be5ab0abe15eeR7 to set the project field.

Reply:
never mind then!